plutus-chain-index/app/Main.hs

```diff
@@ -135,6 +137,17 @@ main = do
       putStrLn "\nChain Index config:"
       print (pretty config)
 
+      -- The printed slot number is only half helpful.
+      -- The primary purpose of this query is to get the first response of the node for potential errors before opening the DB and starting the chain index.
+      -- See #69.
+      putStrLn "\nQuery the tip of the local node:"
```

Comment:
```suggestion
      putStr "\nThe tip of the local node: "
```

Reply:
Thanks! I've updated the message as suggested.